kinds/types/release_metadata.go

```diff
@@ -8,17 +8,18 @@ import (
 // ReleaseMetadata holds the metadata about a specific release, including addons and
 // their versions.
 type ReleaseMetadata struct {
-	Versions       map[string]string
-	K0sSHA         string
-	K0sBinaryURL   string
-	Artifacts      map[string]string // key is the artifact name, value is the URL it can be retrieved from
-	Images         []string
-	K0sImages      []string                // deprecated (still used by airgap-builder), use Images instead
-	Configs        v1beta1.Helm            // always applied
-	BuiltinConfigs map[string]v1beta1.Helm // applied if the relevant builtin addon is enabled
-	Protected      map[string][]string
+	Versions     map[string]string
+	K0sSHA       string
+	K0sBinaryURL string
+	Artifacts    map[string]string // key is the artifact name, value is the URL it can be retrieved from
+	Images       []string
+	K0sImages    []string     // deprecated (still used by airgap-builder), use Images instead
+	Configs      v1beta1.Helm // always applied
 
 	// Deprecated: AirgapConfigs exists for historical compatibility and should not
-	// be used. This field has been replaced by the BuiltinConfigs field.
+	// be used. This field has been replaced by the Configs field.
 	AirgapConfigs k0sv1beta1.HelmExtensions
+	// Deprecated: BuiltinConfigs exists for historical compatibility and should not
+	// be used. This field has been replaced by the Configs field.
+	BuiltinConfigs map[string]v1beta1.Helm // applied if the relevant builtin addon is enabled
```

Comment:
I think the replacement listings here are incorrect, we're instead just not using it at all
we should probably drop Configs too

Reply:
something to fix in the future